src/components/document-metadata-modal.yml

```diff
@@ -10,6 +10,9 @@ titleRequired:
 descriptionTooLong:
   en: The description cannot have more than {maxChars, plural, =1 {# character} other {# characters}}
   de: Die Beschreibung darf nicht länger als {maxChars} Zeichen sein
+tagsPlaceholder:
+  en: Enter keywords
+  de: Stickwörter angeben
```

Comment:
That would be "embroidery words" :rofl: We need "Stic**h**wörter"

Reply:
😂 oh too bad! 👵